static/js/tasks_browse.js

```diff
@@ -291,8 +291,14 @@ $(document).ready(function() {
         showAssignWorkerModal()
     });
 
-    $('#delete-tasks-modal .cancel-delete').on('click', function() {
+    // reset the local variable as well as the state
+    function retSelectedTask() {
```

Comment:
Would a more appropriate name for this method be `resetSelectedTask()`?

Reply:
It is. It was a typo :(